src/status_im2/contexts/onboarding/identifiers/profile_card/style.cljs

```diff
@@ -66,4 +66,5 @@
           {:margin-top 2})))
 
 (def emoji-hash
-  {:margin-top 12})
+  {:margin-top  12
+   :line-height 20.5})
```

Comment:
are you sure about `.5` ?

Reply:
Yeah, the lineHeight of 20.5 matches the designs spacing and height. 

The design uses regular images in a row for the emoji hash while we use text for this, and the lineHeight of the text used initially was 21.75 (`paragraph-1` in typography).